viz/package.json

```diff
@@ -1,62 +1,62 @@
 {
   "name": "geojson",
-  "version": "0.0.0",
-  "livscode-file://vscode-app/c:/Users/jins176/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.htmlcense": "MIT",
-  
+  "version": "0.1.0",  
   "scripts": {
+    "prepare": "cd .. && husky install",
     "start-local": "webpack-dev-server --env.local --progress --hot --open",
     "start": "webpack-dev-server --progress --hot --open "
   },
   "dependencies": {
-    "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1",
-    "@flowmap.gl/data": "^8.0.0-alpha.25",
-    "@flowmap.gl/layers": "^8.0.0-alpha.25",
-    "@mui/icons-material": "^5.6.2",
-    "@mui/material": "^5.6.4",
-    "@turf/turf": "^5.1.6",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-es2015": "^6.24.1",
-    "bootstrap": "^5.3.0",
-    "buffer": "^6.0.3",
-    "chart.js": "^3.5.0",
-    "core-js": "^3.31.1",
```

Comment:
Note that I am really hacking away here (unsuccessfully), so some guidance from Sichen would be awesome here

Reply:
Does setting the version numbers of the packages to null work on your system?

Reply:
Going to try adding back in these versions and pinning Node version. Should get a build going I hope, but I still have some open questions about preparing data for viz...